hasher-matcher-actioner/docs/docs/installation/cognito.md

```diff
@@ -0,0 +1,39 @@
+---
+id: cognito
+title: Cognito User Pools for Authentication 
+---
+
+## Terraforming a User Pool
+
+This starts after you have checked out a release of HMA and unzipped it. Check out this [section in the installation](installation/#checking-out-a-release) page.
+
+```sh
+$ cd ThreatExchange-HMA-v0.1.1/hasher-matcher-actioner/
+$ cd terraform/authentication-shared/
+$ cp terraform.tfvars.example terraform.tfvars
+```
+
+Open `terraform.tfvars` in an editor and set the value of `organization` to a short string that denotes the organization you work for. It may use the name of the organization, or it may not. It is used as a part of the name for s3 buckets which are expected to be globally unique. Use a short, all-smallcase string without any hyphens or underscores.
```

Comment:
nit: ~~organization you work for~~ your organization. terraform uses this try guarantee uniqueness beyond `prefix` where required by aws.